Tella/Data/VaultManager/VaultManager.swift

```diff
@@ -80,7 +80,7 @@ class VaultManager : VaultManagerInterface, ObservableObject{
 
     func loadVaultFileToURL(file vaultFile: VaultFileDB) -> URL? {
 
-        let tmpFileURL = createTempFileURL(pathExtension: vaultFile.fileExtension)
+        let tmpFileURL = createTempFileURL(fileName: vaultFile.name ,pathExtension: vaultFile.fileExtension)
         
```

Comment:
I think it is more readable to pass directly the `vaultFile`  as a parameter instead of passing multiple params this keeps the method cleaner and easy to understand

Reply:
Yeah, it makes sense! The createTempFileURL method is reusable and is utilized in various parts of the app. In some instances, we only have the fileName or path instead of the VaultFileDB object, which is why we limit the parameters to these two variables. Modifying the method to accept only VaultFileDB would require applying these changes to other methods as well, extending beyond the scope of this PR.